src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2289,226 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("decodingtrust_stereotype_bias")
+def get_stereotype_bias_spec(task: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_stereotype_prompts_scenario"
+                   ".DecodingTrustStereotypePromptsScenario",
+        args={},
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=25, max_tokens=150, temperature=1)
+
+    return RunSpec(
+        name=f"stereotype_bias",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_stereotype_bias_metric_specs(),
+        groups=["decoding_trust", "stereotype_bias"],
+    )
+
+
+@run_spec_function("decodingtrust_adv_robustness")
+def get_adv_robustness_spec(task: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_adv_robustness_scenario.AdvRobustnessScenario",
+        args={"glue_task": task},
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=1, max_tokens=16, temperature=0)
+
+    return RunSpec(
+        name=f"adv_robustness:task={task}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_exact_match_metric_specs(),
+        groups=["decoding_trust", "adv_robustness"],
+    )
+
+
+@run_spec_function("decodingtrust_adv_demonstration")
+def get_adv_demonstration_spec(perspective: str, data: str, demo_name: str, description: str) -> RunSpec:
```

Comment:
As discussed in the `run_specs_decodingtrust.conf` comments - for this function and other functions below, make parameters optional with default values where applicable.